server/data/quotes.json

```diff
@@ -571,6 +571,14 @@
     "author": "Vivien Leigh",
     "sentence": "I never found accents difficult, after learning languages."
   },
+  {
+    "author": "Waren Gonzaga",
+    "sentence": "Learning foreign languages is like learning computer programming languages."
+  },
+  {
+    "author": "Waren Gonzaga",
+    "sentence": "The more language you know, the better."
```

Comment:
@WarenGonzaga good with the first quote, not sure about this one though?

Reply:
@jayehernandez basically and indirectly duplicate?